pikaur/args.py

```diff
@@ -53,11 +53,13 @@ def get_pikaur_bool_opts() -> ArgSchema:
         (None, 'nodiff', PikaurConfig().review.NoDiff.get_bool()),
         (None, 'rebuild', None),
         (None, 'dynamic-users', PikaurConfig().build.AlwaysUseDynamicUsers.get_bool()),
+        (None, 'skip-failed-build', PikaurConfig().build.SkipFailedBuild.get_bool()),
```

Comment:
please also add test for this option as well, you could create/use 'broken' PKGBUILD inside `pikaur_test` directory for that